app/data/request_types.json

```diff
@@ -552,26 +563,26 @@
   {
     "value": "UC",
     "short_desc": "Unlimited Liability Company ",
-    "text": "BC to Unlimited Liability Company Conversion",
+    "text": "BC Limited to Unlimited Liability Company Conversion",
     "class_type_cd": "CORP",
     "legacy_cd": "UC",
     "additional_info": "",
     "source_application": "COLIN",
-    "entity_type_cd": "UC",
+    "entity_type_cd": "UL",
```

Comment:
A couple of comments about this.

1. I _assumed_ it should be UL (the code for a regular ULC) because all the other CNV entries in here apply a code for a regular company type. However, it's possible that all CNV entries are wrong and this value should be for a _continued in_ company type (eg, BECT, CCCT, CTC, CT and ULCT). Can you test that usable conversion NRs are created when they is changed by this list (drop-down menu)? I can help. Let me know.

2. Although this field is labeled "entity type code", this is actually a namex type code. For example, code "CR" is used on line 571 for a BC Limited. Entities uses a different code (BC). Another example: on line 847, code 'BC" is used for a Benefit Company. Entities uses a different code (BEN). I am not asking you to rename this field, but you should be aware of the difference I just explained.

Reply:
Steven and I tested together.
- CCV worked.
- UC worked.
- BECC didn't work, but creating a new NR of type BECC also didn't work, so the problem likely lies somewhere else (maybe Business Edit UI).